examples/firecrawl_default_config_example.ts

```diff
@@ -0,0 +1,166 @@
+/**
```

Comment:
same comment as above re: `createReactAgent` - and it will also make it easier to highlight the differences between custom config / default config

Reply:
Changed it to createReactAgent